boefjes/boefjes/plugins/kat_external_db/normalize.py

```diff
@@ -17,6 +17,8 @@
 IP_ADDRESS_ITEM_PATH = ["address"]
 DOMAIN_LIST_PATH = ["domains"]
 DOMAIN_ITEM_PATH = ["name"]
+INDEMNIFICATION_ITEM_PATH = ["indemnification_level"]
+DEFAULT_INDEMNIFICATION_LEVEL = 3
```

Comment:
I am not sure we should have a default indemnification level of 3. Wouldn't 2 be more appropriate?

Reply:
It used to be 3 already, this PR only adds it from an external database